packages/react-server-cli/src/coreCssMiddleware.js

```diff
@@ -0,0 +1,13 @@
+
+module.exports = function(pathToStatic) {
```

Comment:
No, you're right.  A global would be far worse.  I wonder about a static method that sets the prefix?

Anyway, not something that needs to hold this PR up!